src/components/StatisticsModal/StatisticsModal.js

```diff
@@ -0,0 +1,121 @@
+import styled from 'styled-components';
+import ScreenReaderOnly from '../ScreenReaderOnly';
+import { GrFormClose } from 'react-icons/gr';
+
+export default function StatisticsModal({
+  divenumber,
+  location,
+  city,
+  country,
+  date,
+  buddy,
+  typeOfDive,
+  timeIn,
+  timeOut,
+  bottomTime,
+  maxDepth,
+  notes,
+  onStatisticsToggle,
+}) {
+  return (
+    <Background>
+      <Card>
+        <CardHead>
+          <div>
+            <p>{city}</p>
+            <p>{country}</p>
+            <p>{location}</p>
+          </div>
+          <BoxDateDelete>
+            <ButtonClose onClick={onStatisticsToggle}>
+              <GrFormClose />
+              <ScreenReaderOnly>close</ScreenReaderOnly>
+            </ButtonClose>
+            <p>{date}</p>
+          </BoxDateDelete>
+        </CardHead>
+
+        <CardDetails>
+          <p>time in: </p>
+          <p>{timeIn}</p>
+          <p>time out: </p>
+          <p>{timeOut}</p>
```

Comment:
maybe this could be span elements to be able to style them in one line and leave out the time in and time out. You could just but a " - " in between, just to bring some excitement to the structure of the card haha :)